### ISSUE TITLE
Rectification Bug? 

### ISSUE THREAD

**Issue by sanjayseth** [wikimaps-dev/mapwarper#60]
Let me know what kind of feedback is helpful, based off of where you are in the development of this tool. 

I'll share a potential bug I've run into today, in case you are at the place where this feedback is useful. Couldn't reverse engineer, but maybe you can see why it's happening. In both of the cases below, I have run the warper from the Rectify tab and from the Crop tab. I get feedback that the map has been rectified. But you can see from the item description that it is not recognized as rectified - and the preview tab will displays an error that "you'll have to rectify a map before you can view it." 

http://warper.wmflabs.org/maps/295

![image](https://cloud.githubusercontent.com/assets/7000560/11725418/9cfc26ea-9f47-11e5-9669-74b300819896.png)

![image](https://cloud.githubusercontent.com/assets/7000560/11725430/a8efbe9e-9f47-11e5-945a-12439e86be93.png)

**Comment by timwaters** [comment]
there does indeed seem like there's something going wrong here

```
I, [2016-03-21T17:07:22.779252 #15975]  INFO -- : gdal translate
I, [2016-03-21T17:07:23.351679 #15975]  INFO -- : gdal_translate -a_srs '+init=epsg:4326' -of VRT /home/warperdata/src/T-1533-A.jpg.tif_masked 295_temp.vrt  -gcp 5642.13348214286, 3707.41741071428, -73.966667, 41.916667 -gcp 2866.59642857143, 2024.59285714286, -73.983333, 41.916667 -gcp 8428.60948660714, 5378.12042410714, -73.95, 41.916667 -gcp 8430.38337053569, 5379.00736607143, -73.95, 41.916667
I, [2016-03-21T17:07:23.442353 #15975]  INFO -- : gdalwarp    -rn    -dstalpha  -srcnodata '17 17 17'  -s_srs 'EPSG:4326' dst/295.tif -co TILED=YES -co COMPRESS=LZW
E, [2016-03-21T17:07:23.442742 #15975] ERROR -- : Error gdal warp scriptERROR 1: Attempt to create 13589x-2147483648 dataset is illegal,sizes must be larger than zero.

E, [2016-03-21T17:07:23.442876 #15975] ERROR -- : output = Creating output file that is 13589P x -2147483648L.

I, [2016-03-21T17:07:23.497435 #15975]  INFO -- : gdaladdo -r average 295.tif 2 4 8 16 32 64
E, [2016-03-21T17:07:23.497943 #15975] ERROR -- : Error gdal overview scriptERROR 4: `295.tif' does not exist in the file system,
and is not recognised as a supported dataset name.

```

```
gdalinfo T-1533-A.jpg.tif_masked
Driver: GTiff/GeoTIFF
Files: T-1533-A.jpg.tif_masked
Size is 11023, 7947
Coordinate System is `'
Image Structure Metadata:
  COMPRESSION=DEFLATE
  INTERLEAVE=PIXEL
Corner Coordinates:
Upper Left  (    0.0,    0.0)
Lower Left  (    0.0, 7947.0)
Upper Right (11023.0,    0.0)
Lower Right (11023.0, 7947.0)
Center      ( 5511.5, 3973.5)
Band 1 Block=11023x1 Type=Byte, ColorInterp=Red
  Overviews: 5512x3974, 2756x1987, 1378x994, 689x497, 345x249, 173x125
Band 2 Block=11023x1 Type=Byte, ColorInterp=Green
  Overviews: 5512x3974, 2756x1987, 1378x994, 689x497, 345x249, 173x125
Band 3 Block=11023x1 Type=Byte, ColorInterp=Blue
  Overviews: 5512x3974, 2756x1987, 1378x994, 689x497, 345x249, 173x125

```

**Comment by timwaters** [comment]
Okay, seems like this wasn't an issue with the platform but with the points.

The point #4 was on almost the same spot. 

![screenshot from 2016-03-21 17 14 51](https://cloud.githubusercontent.com/assets/48941/13927407/8354d3d0-ef89-11e5-8f07-cebd925af7a0.png)

so I removed this, leaving with 3 points, all very neatly in a line

```
E, [2016-03-21T17:15:44.677576 #15975] ERROR -- : Error gdal warp scriptERROR 1: Failed to compute GCP transform: Transform is not solvable

```

and as the error message implies, it needed better solvable control points, I added one and it works now

Control points need to be somewhat spread out over the map, or at least not all on a single line for the math magics to happen.  

![screenshot from 2016-03-21 17 25 47](https://cloud.githubusercontent.com/assets/48941/13927536/fff604d6-ef89-11e5-8772-9ff83e0e682b.png)
